README.md

```diff
@@ -106,6 +107,13 @@ Run all tests with `deno test`:
 $ deno test
 ```
 
+## Datastores
+
+If your app needs to store any data, a datastore would be the right place for
+that. For an example of a datastore, see `datastores/sample_datastore.ts`.
+Remember to add the `datastores:write`/`datastores:read` scopes and uncomment
+the `datastores` key in your manifest file if you are using a datastore.
```

Comment:
All scaffolding in place should be part of the working application, so we can add this paragraph so long as we also add the corresponding working example. After the working example is in place, this paragraph will likely need to be tweaked a bit.

We should also place this _before_ the **Testing** section.

Additionally, I've never used datastores, so what is the difference (if any) between using a datastore for development and using it for production?